src/components/layouts/ProjectFeedLayout/index.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import Link from '../../atoms/Link';
+import { getComponent } from '../../components-registry';
+import { getBaseLayoutComponent } from '../../../utils/base-layout';
+import { mapStylesToClassNames as mapStyles } from '../../../utils/map-styles-to-class-names';
+
+export default function ProjectFeedLayout(props) {
+    const { page, site } = props;
+    const BaseLayout = getBaseLayoutComponent(page.baseLayout, site.baseLayout);
+    const { title, topSections = [], bottomSections = [], pageIndex, baseUrlPath, numOfPages, items, projectFeed } = page;
+    const projectFeedWidth = projectFeed?.styles?.self?.width || 'wide';
+    const projectFeedJustifyContent = projectFeed?.styles?.self?.justifyContent || 'center';
+    const ProjectFeedSection = getComponent('ProjectFeedSection');
+    const pageLinks = PageLinks({ pageIndex, baseUrlPath, numOfPages });
+
+    return (
+        <BaseLayout page={page} site={site}>
+            <main id="main" className="layout page-layout">
+                {title && (
+                    <div className={classNames('flex', 'py-12', 'lg:py-16', 'px-4', mapStyles({ justifyContent: projectFeedJustifyContent }))}>
+                        <h1
+                            className={classNames('w-full', mapMaxWidthStyles(projectFeedWidth), page?.styles?.title ? mapStyles(page?.styles?.title) : null)}
+                            data-sb-field-path="title"
+                        >
+                            {title}
+                        </h1>
+                    </div>
+                )}
+                {renderSections(topSections, 'topSections')}
```

Comment:
@TomasBankauskas rename the function to `Sections` with capital `S` and use it as component:
```
<Sections sections={topSections}/>
```